config/config.exs

```diff
@@ -28,3 +28,7 @@ use Mix.Config
 # here (which is why it is important to import them last).
 #
 #     import_config "#{Mix.env()}.exs"
+
+config :active_proxy,
+  timeout: 100,
+  node1_address: "159.203.44.11"
```

Comment:
This should be a list key instead of a indexed key lol
```
upstream_hosts: [
  "159.203.44.11"
]
```

Reply:
I think we talked about change address during runtime so I assumed named addresses would be best. Not sure if we still want to do that.

Reply:
Purposal for actual storage: store active/inactive data in a persistent KV store instead because it's more flexible. Imagine the case where we decide to have dynamic upstream being added and the proxy restarts. That state is lost and the proxy goes back to relying on the static config. Whereas with the KV store we'd have that state saved. Obviously, this is out of scope for this PR but we should consider this for future active/inactive storage. 

For this PR I say you just make a decision of a list or a named active/inactive hosts.